nav2_collision_monitor/include/nav2_collision_monitor/types.hpp

```diff
@@ -27,8 +27,10 @@ struct Velocity
 
   inline bool operator<(const Velocity & second) const
   {
-    const double first_vel = x * x + y * y;
-    const double second_vel = second.x * second.x + second.y * second.y;
+    const double first_vel = x * x + y * y + tw * tw;
+    const double second_vel = second.x * second.x + second.y * second.y + second.tw * second.tw;
+    // In Collision Monitor twists will change proportionally to linear velocities,
```

Comment:
I'm not sure you need the comment to justify it - if you add a comment at all it should just be explaining what its doing / the intention (and the fact that this resolves rotation in place issues)

Reply:
Okay, I've [updated](https://github.com/ros-planning/navigation2/pull/3177/commits/32bcf9e603293b973c44664266bfb4b35c339a04) the comment intention